modules/auxiliary/scanner/discovery/ipv6_neighbor.rb

```diff
@@ -74,7 +74,7 @@ def run_batch(hosts)
         end
       end
 
-      etime = ::Time.now.to_f + (hosts.length * 0.05)
+      etime = ::Time.now.to_f + 1
```

Comment:
Thinking we should perhaps keep the time scaling but with a minimum value of one like you're using here. That would ensure we're always waiting at least 1 second.

```suggestion
      etime = ::Time.now.to_f + [1, hosts.length * 0.05].max
```